node/src/actors/chain_manager/mod.rs

```diff
@@ -484,6 +485,21 @@ impl ChainManager {
                 chain_info.highest_block_checkpoint = beacon;
                 chain_info.highest_vrf_output = vrf_input;
 
+                // Store the ARS and the order of the keys
+                let trs = reputation_engine.trs();
+                let current_ars = reputation_engine
+                    .ars()
+                    .active_identities()
+                    .cloned()
+                    .sorted()
+                    .collect();
+                let alt_keys = &self.chain_state.alt_keys;
+
+                let ordered_alts: Vec<Bn256PublicKey> = alt_keys.get_rep_ordered_bn256_list(trs);
+                // last ars with previous block ars info
+                self.chain_state.last_ars = current_ars;
+                self.chain_state.last_ars_ordered_keys = ordered_alts;
```

Comment:
last_ars is also ordered, so that names could be misunderstanding. Because the main difference is that one are bls and the other secp??

Reply:
the truth is last_ars does not need to be actually sorted. I will fix that.